Documentation/Available-Snapshot-Strategies.md

```diff
@@ -40,6 +40,7 @@ If you'd like to submit your own custom strategy, see [Contributing](../CONTRIBU
       - [`.recursiveDescription`](#recursivedescription-3)
   - [`URLRequest`](#urlrequest)
       - [`.raw`](#raw)
+      - [`.curlRepresentation`](#curlRepresentation)
 
```

Comment:
Wanna move this before `.raw`? We don’t document it but our one go-to for organization is alphabetizing.

Reply:
Not a problem, this is done 😄